packages/react/unstable-shared-subset.experimental.js

```diff
@@ -33,6 +33,7 @@ export {
   SuspenseList as unstable_SuspenseList,
   unstable_useOpaqueIdentifier,
   unstable_getCacheForType,
+  unstable_useRefresh,
```

Comment:
This should not be included in this file. Which means that the error the dispatcher throws should never be reachable, but worth keep in case something is misconfigured or tries use internals.

This ensures that statically, we can say that a shared/server component can't import this at all so there's no risk of accidentally using it and it's a signal that a client component is needed.